runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -290,6 +291,511 @@ benchmarks! {
+         (T::StorageBucketsPerBagValueConstraint::get().max() as u32);
+
+        let c in 1 .. MAX_BYTES_METADATA; //max bytes for rationale
+
+        let (
+            channel_id,
+            group_id,
+            lead_account_id,
+            curator_id,
+            curator_account_id
+        ) =
+        setup_worst_case_scenario_curator_channel::<T>(
+            a,
+            b,
+            T::DistributionBucketsPerBagValueConstraint::get().max() as u32, true).unwrap();
+
+        let origin = RawOrigin::Signed(curator_account_id);
+        let actor = ContentActor::Curator(group_id, curator_id);
+        let storage_buckets_num_witness =
+            storage_buckets_num_witness::<T>(channel_id)?;
+        let rationale = vec![1u8].repeat(c as usize);
+
+    }: _ (origin,
+        actor,
+        channel_id,
+        storage_buckets_num_witness,
+        a.into(),
+        rationale.clone())
+    verify {
+        assert_last_event::<T>(
+            Event::<T>::ChannelDeletedByModerator(
+                actor,
+                channel_id,
+                rationale
+            ).into()
+        );
+    }
+
+    set_channel_visibility_as_moderator{
+
+        let a in 1 .. MAX_BYTES_METADATA; //max bytes for rationale
+
+        let (
+            channel_id,
+            group_id,
+            _,
+            curator_id,
+            curator_account_id
+        ) =
+        setup_worst_case_scenario_curator_channel::<T>(
+            T::MaxNumberOfAssetsPerChannel::get(),
+            T::StorageBucketsPerBagValueConstraint::get().max() as u32,
+            T::DistributionBucketsPerBagValueConstraint::get().max() as u32,
+            true
+        ).unwrap();
+
+        let origin = RawOrigin::Signed(curator_account_id);
+        let actor = ContentActor::Curator(group_id, curator_id);
+
+        let rationale = vec![0u8].repeat(a as usize);
+
+    }: _ (origin, actor, channel_id, true, rationale.clone())
+    verify {
+
+        assert_last_event::<T>(
+            Event::<T>::ChannelVisibilitySetByModerator(
+                actor,
+                channel_id,
+                true,
+                rationale,
+            ).into());
+    }
+
+    delete_video_assets_as_moderator_with_assets {
+
+        let a in 1 .. T::MaxNumberOfAssetsPerVideo::get(); //max objs number
+
+        let b in (T::StorageBucketsPerBagValueConstraint::get().min as u32) ..
+         (T::StorageBucketsPerBagValueConstraint::get().max() as u32);
+
+        let c in 1 .. MAX_BYTES_METADATA; //max bytes for rationale
+
+        let (
+            channel_id,
+            group_id,
+            _,
+            curator_id,
+            curator_account_id
+        ) =
+        setup_worst_case_scenario_curator_channel::<T>(
+            T::MaxNumberOfAssetsPerChannel::get(),
+            b,
+            T::DistributionBucketsPerBagValueConstraint::get().max() as u32,
+            false
```

Comment:
This cannot be set to true, because this sets transfer status to `PendingTransfer`.
When creating a video, `ensure_has_no_active_transfer` is called and returns an error if 
`transfer_status` is different from `ChannelTransferStatus::NoActiveTransfer.`

Reply:
Ok, I guess you could change the transfer status after the video is added, but it's not really that important, it probably has negligible effect on the weight (if any)